### PR TITLE
INDY-1083: do not allow overriding of existing claim defs

### PR DESCRIPTION
**Pull request by ashcherbakov**
Signed-off-by: ashcherbakov <alexander.sherbakov@dsr-company.com>

### CODE REVIEW

File: indy_node/server/domain_req_handler.py [hunk 1]
```diff
@@ -214,6 +200,40 @@ def _validateAttrib(self, req: Request):
...
+                                       'name {} and version {}'
+                                       .format(identifier, schema_name, schema_version))
+
+    def _validate_claim_def(self, req: Request):
+        # we can not add a Claim Def with existent ISSUER_DID
+        # sine a Claim Def needs to be identified by seqNo
+        identifier = req.identifier
+        operation = req.operation
+        schema_ref = operation[REF]
+        claim_def, _, _, _ = self.getClaimDef(
```
Comment (lovesh) [inline indy_node/server/domain_req_handler.py]: An issuer should be allowed to have multiple claim definitions for the same schema, it's just that they need to be for different signature schemes

File: indy_node/server/domain_req_handler.py [hunk 2]
```diff
@@ -214,6 +200,40 @@ def _validateAttrib(self, req: Request):
...
+            author=identifier,
+            schemaSeqNo=schema_ref
+        )
+        if claim_def:
+            raise InvalidClientRequest(identifier, req.reqId,
+                                       '{} can have one and only one CLAIM_DEF with '
+                                       'issuer DID (identifier) {} and schema ref {}'
```
Comment (lovesh) [inline indy_node/server/domain_req_handler.py]: This should read 'signature scheme {} and schema ref {}'

File: indy_node/server/domain_req_handler.py (hunk 1 above)
Comment (ashcherbakov) [reply indy_node/server/domain_req_handler.py]: Done

File: indy_node/server/domain_req_handler.py (hunk 2 above)
Comment (ashcherbakov) [reply indy_node/server/domain_req_handler.py]: Done

**Pull request merged**